aries-site/src/pages/components/tag.mdx

```diff
@@ -1,144 +1,122 @@
 
-- In some hybrid use cases, it can be valid to have a mixture of tags with name-value and others with just value.
+## Anatomy
 
-### Creating name-value pair tags
+<Example
+  caption="Diagram illustrating the four parts composing tag."
+  pad="small"
+  previewWidth="large"
+  height={{ min: 'small' }}
+>
+  <TagAnatomy />
+</Example>
 
-The experience for creating a Name-Value pair tag occurs in 2 steps: Create the name and then create the value. Alternatively, select an existing name and create a new value.
 
-## Assigning tags
+| Label | Region        | Purpose                                                      | Required | Notes                                                       |
+| :---: | ------------- | ------------------------------------------------------------ | :------: | ----------------------------------------------------------- |
+| **1** | **Container** | Controls padding, border & background color.              | Required | Behavior and style will change depending on tag type.                                                        |
+| **2** | **Name**      | Descriptive label or category that identifies what kind of data is stored in the associated value. For example “Country” or “Color”.             | Optional | Up to 128 characters max.                     |
+| **3** | **Value**     | The specific descriptive data associated with an item.  For example “USA” or “Green”. | Required | Up to 256 characters max. |
+| **4** | **Close Icon Button**      | Used to indicate that tag is removable. | Optional | Required for only removable tags.  |
 
-Currently, assigning tags occurs within an editor experience in a [modal](/components/layer#side-drawer-modal) or [wizard](/templates/wizard). This helps to focus the task for the user. Tags are first selected and/or created, then assigned to the resource.
 
-### Assigning value-only tags
+## Types
 
-<Example
-  code={[
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tag/CreateTagSimple.js',
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tag/CreateTag/AppContainer.js',
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tag/CreateTag/TagPageHeader.js',
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tag/CreateTag/TagResults.js',
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tag/CreateTag/utils.js',
-  ]}
-  github="https://github.com/grommet/hpe-design-system/tree/master/aries-site/src/examples/components/tag/CreateTagSimple.js"
-  docs="https://v2.grommet.io/tag?theme=hpe#props"
-  showResponsiveControls={['laptop', 'mobile']}
-  template
->
-  <CreateTagSimple />
-</Example>
+### Tag 
```

Comment:
Seems like this should just be "Static", Tag is the thing and static is the type. Then the other types build off of static.

If so, rename this to "Static" and remove the current section called "Static".